files/zh-cn/web/api/eventsource/message_event/index.md

```diff
@@ -57,7 +57,7 @@ evtSource.addEventListener("message", (e) => {
 ```
 
 ### onmessage 的等价写法
-
+仅仅在接收无类型消息的时候，才会通过onmessage处理函数。
```

Comment:
Hi @shylockWu. We only "[translate](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md)" en-US documents, and this line is not existed [here](https://github.com/mdn/content/blob/03ba2dad6861d4d58be6be41c2401dab7a604a75/files/en-us/web/api/eventsource/message_event/index.md?plain=1#L62-L71). I'm sorry we could not merge this (: